Task_DEV_4/Task_DEV_4/Circle.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+namespace Task_DEV_4
+{
+    class Circle : Figure
+    {
+        private const string MESSAGE_ERROR_FIRST = "Radius cannot be less or equals than 0.";
```

Comment:
Bad variable name. Name it smth like INVALID_RANGE_ERROR_MESSAGE. Name has to reflect sense.

Reply:
Less or equal. Without s.

Reply:
Done.